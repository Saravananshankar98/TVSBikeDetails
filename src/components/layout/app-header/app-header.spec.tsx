import { render } from '@testing-library/react';
import React from 'react'
import { TestContextProvider } from '../../../test/test-container';
import AppHeader from './app-header';

describe("CardBikes", () => {
    let baseElement: any, container, screen;
    beforeEach(() => {
      ({ container, baseElement, ...screen } = render(
        <AppHeader/>,
        {
          wrapper: TestContextProvider,
        }
      ));
    });
  
    it("Matches Snapshot", () => {
      expect(baseElement).toMatchSnapshot();
    });
  });