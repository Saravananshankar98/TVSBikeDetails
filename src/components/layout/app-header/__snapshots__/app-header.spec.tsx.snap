// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardBikes Matches Snapshot 1`] = `
<body>
  <div>
    <div
      class="MuiBox-root css-0"
    >
      <header
        class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionFixed mui-fixed css-11b3ww9-MuiPaper-root-MuiAppBar-root"
      >
        <div
          class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-uapk2y-MuiToolbar-root"
        >
          <div
            class="css-qlqe4h"
          >
            <div
              class="css-1iz4iln"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="SearchIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                />
              </svg>
            </div>
            <div
              class="MuiInputBase-root MuiInputBase-colorPrimary css-1kcggdq-MuiInputBase-root"
            >
              <input
                aria-label="search"
                class="MuiInputBase-input css-yz9k0d-MuiInputBase-input"
                placeholder="Searchâ€¦"
                type="text"
                value=""
              />
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-grid-md-12 css-1azvbld-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-grid-xs-1 css-4a9a00-MuiGrid-root"
            >
              <h5
                class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
              >
                 test Ride
              </h5>
            </div>
            <div
              class="MuiGrid-root MuiGrid-grid-xs-1.5 css-1tk4wtl-MuiGrid-root"
            >
              <h5
                class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
              >
                 Book Vehicle
              </h5>
            </div>
            <div
              class="MuiGrid-root MuiGrid-grid-xs-0.4 css-fadvaw-MuiGrid-root"
            >
              <div
                class="MuiBox-root css-0"
              >
                <button
                  aria-haspopup="true"
                  aria-label="account of current user"
                  class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeLarge css-1pz7awu-MuiButtonBase-root-MuiIconButton-root"
                  tabindex="0"
                  type="button"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="AccountCircleIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 4c1.93 0 3.5 1.57 3.5 3.5S13.93 13 12 13s-3.5-1.57-3.5-3.5S10.07 6 12 6zm0 14c-2.03 0-4.43-.82-6.14-2.88C7.55 15.8 9.68 15 12 15s4.45.8 6.14 2.12C16.43 19.18 14.03 20 12 20z"
                    />
                  </svg>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </div>
            </div>
          </div>
        </div>
      </header>
    </div>
  </div>
</body>
`;
